test/Dialect/MSFT/location.mlir

```diff
@@ -61,7 +61,7 @@ msft.physical_region @region1, [
 // TCL-LABEL: proc regions_config
 msft.module @regions {} () -> () {
   msft.instance @module @Foo() {
```

Comment:
Actually, the attributes dict can be deleted from both these ops. I missed them after s/globalRef/hierpath/.

Reply:
#6161 fixed this.